_posts/2017/09/2017-09-19-rfc.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+authors: ["Kate Hertweck"]
+title: "Request for Comment: Share your thoughts on the future of The Carpentries"
+date: 2017-09-19
+time: "15:00:00"
+category: ["Steering Committee", "Community", "Data Carpentry"] 
+---
+
+**We are now accepting comments on plans related to The Carpentries!**
```

Comment:
accepting -> requesting 
We want to make it clear that we want comments. Also it aligns since they are called 'requests for comments'